### PR TITLE
feat(Less): Add accessible variant for mid-gray-dark on gray-medium

### PR DESCRIPTION
**Pull request by kosanna**

### CODE REVIEW

**Comment by markdalgleish** [comment]
Just a heads up that I renamed this PR from a "refactor" to a "feat" — since this changes the public API, it's a feature release. Also, refactors don't trigger a release, if I remember correctly.

**Comment by markdalgleish** [comment]
Just noticed the readme hasn't been updated with the new colour. Would you be able to add that?

**Comment by kosanna** [comment]
Yup, will update

**Comment by kosanna** [comment]
Updated

**Comment by markdalgleish** [comment]
I don't want to block this PR over it, but we might need to rethink our naming convention around accessible variants. The current approach isn't very scalable at all.

**Comment by kosanna** [comment]
They are all gray. Aren't you bugged with css `color` property, not `colour`?

**Comment by markdalgleish** [comment]
Sorry, my comment was aimed at your spelling in this case—you've done the right thing and remained consistent.

File: theme/palette/accessible-variants.less [hunk 1]
```diff
@@ -1,5 +1,7 @@
 // Accessible variants
 @sk-mid-gray-on-white: #959595;
+@sk-mid-gray-dark-on-gray-medium: #646464;
```
Comment (michaeltaranto) [inline theme/palette/accessible-variants.less]: As discussed this should be `@sk-mid-gray-dark-on-gray-light` as that is what the [background is set to](https://github.com/seek-oss/seek-style-guide/blob/master/theme/palette/elements.less#L10).

**Pull request merged**